### PR TITLE
[NETBEANS-1152] Moving modules from groovy cluster into a subdirectory

### PR DESCRIPTION
**Pull request by geertjanw**

### CODE REVIEW

File: groovy/groovy.antproject/build.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
     under the License.
 
 -->
-<project basedir="." default="netbeans" name="groovy.antproject">
+<project basedir="." default="build" name="groovy/groovy.antproject">
```
Comment (matthiasblaesing) [inline groovy/groovy.antproject/build.xml]: Could you please explain the reason for changing the default target and the name?

**Comment by geertjanw** [comment]
Seems to have passed all checks and balances. :-) Maybe we should simply merge this?

**Comment by geertjanw** [comment]
Thanks for identifying this, will investigate.

File: groovy/groovy.antproject/build.xml (hunk 1 above)
Comment (geertjanw) [reply groovy/groovy.antproject/build.xml]: Read the explanation here: https://github.com/apache/incubator-netbeans/pull/719

**Comment by geertjanw** [comment]
> the change was cosmetic (display in github)

No, that's not true: https://github.com/apache/incubator-netbeans/pull/536

**Comment by matthiasblaesing** [comment]
I did not want to discuss #536 - but quote from #536 (no other reasons are given there):

> Recently two new cons appeared:
> 
>- github isn't willing to show the whole repository - it limits the list to just a thousand files
>- new modules are (hopefully) going to be granted by Oracle and not all of them are of the same importance

this qualifies for me as "no technical need". I can live with the PR, but it needs more work and we are now seeing the fallout.

**Comment by geertjanw** [comment]
Sure, I agree with you. Good this is not merged yet. But is the same problem apparent in the already merged platform reorganization? Probably all this went too fast, apologies for that, and thanks again for spotting the fallout.

**Comment by matthiasblaesing** [comment]
Ok - two observations: the downloads tasks (called from `nbbuild/build.xml`) makes assumptions about the directory layout, that are not valid anymore. The same is true for `VerifyLibsAndLicenses`.

**Comment by geertjanw** [comment]
OK, fair enough. We need to work on these parts before merging anything else related to this change. I want to apologize again for this significant change happening via pull requests rather than much discussion (though to be fair there was a thread by @jtulach on the dev mailing list though admittedly probably too briefly before merging the platform part of this).

**Comment by matthiasblaesing** [comment]
@geertjanw no need to apologize - I think we all have the same aim: Having a great netbeans and an easier time working with the repository. We might not always agree on the "right" way, but the intention is the same :-)

**Comment by geertjanw** [comment]
So, with your (great) changes now merged, can this one be merged too, i.e., there shouldn't be problems anymore with this PR, AFAIK.

**Comment by geertjanw** [comment]
Awesome, thanks, merging!

**Pull request merged**